.github/workflows/publish-command.yml

```diff
@@ -77,10 +77,6 @@ jobs:
         with:
           repository: ${{ github.event.inputs.repo }}
           ref: ${{ github.event.inputs.gitref }}
-      - name: Install Unzip for Databricks
```

Comment:
is this not needed anymore?

Reply:
I think you were looking at only changes from the merge commit. This was brought in from master by a merge conflict, but should not be part of this PR. Do you see this change when you go do https://github.com/airbytehq/airbyte/pull/9742/files (and have no filters applied there)?